README.md

```diff
@@ -1 +1,167 @@
-# witnet-block-relay
\ No newline at end of file
+# witnet-ethereum-block-relay
+
+`witnet-ethereum-block-relay` is an open source implementation of the Block Relay in Witnet.
+
+ DISCLAIMER: this is a work in progress, meaning the contract could be voulnerable to attacks.
+
+## About WBI and the Block Relay
+
+The Bridge nodes interact through the Witnet Bridge Interface, a smart contract in charge of facilitating the communication between Witnet and samrt contract platmforms, such as Ethereum (click [here][wbi] for more information).
```

Comment:
The Bridge Nodes interact through the Witnet Bridge Interface: a smart contract in charge of facilitating the communication between Witnet and smart contract platforms, such as Ethereum.